examples/vmi-fedora-arm64.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    cpu:
+      model: host-passthrough
+    firmware:
+      bootloader:
+        efi: {}
+    devices:
+      autoattachGraphicsDevice: false
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      rng: {}
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - containerDisk:
+      image: registry:5000/kubevirt/fedora-cloud-container-disk-demo:devel
```

Comment:
do we want to explicitly separate the x86 and arm container disk images for examples?

Reply:
We can separate the disk image for different platform, or we can build image for multi-arch.

Reply:
> do we want to explicitly separate the x86 and arm container disk images for examples?

We can separate the disk image for different platform, or we can build image for multi-arch. Any suggestion?